Spigot-Server-Patches/0666-Introduce-beacon-activation-deactivation-events.patch

```diff
@@ -0,0 +1,58 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: Spyridon Pagkalos <spyridon@ender.gr>
+Date: Mon, 25 Jan 2021 23:25:18 +0200
+Subject: [PATCH] Introduce beacon activation/deactivation events
+
+
+diff --git a/src/main/java/net/minecraft/server/TileEntityBeacon.java b/src/main/java/net/minecraft/server/TileEntityBeacon.java
+index f97b27a4751d1fc3287de8dba384ab5639b832b8..f7f43ee3fcbe067451211c7b916a26dc51e38dfa 100644
+--- a/src/main/java/net/minecraft/server/TileEntityBeacon.java
++++ b/src/main/java/net/minecraft/server/TileEntityBeacon.java
+@@ -1,5 +1,7 @@
+ package net.minecraft.server;
+ 
++import io.papermc.paper.event.block.BeaconActivatedEvent;
++import io.papermc.paper.event.block.BeaconDeactivatedEvent;
+ import com.google.common.collect.Lists;
+ import java.util.Arrays;
+ import java.util.Iterator;
+@@ -9,9 +11,7 @@ import java.util.stream.Collectors;
+ import javax.annotation.Nullable;
+ 
+ // CraftBukkit start
+-import org.bukkit.craftbukkit.entity.CraftHumanEntity;
```

Comment:
Don't touch existing imports.

Reply:
That must have been my IDEA messing with them, and I didn't check em 😅 
I'll fix it and comment to mark as ready for merge!